tests/testthat/test-derive_param_wbc_abs.R

```diff
@@ -0,0 +1,125 @@
+test_that("Test adding absolute records for each by group", {
```

Comment:
Do you mind writing out Test1:, Test2:, etc in your file.  The validation report appears slicker and more readable when we use this method.  Not widely adopted yet but I think our new adlb stuff should do this 
![image](https://user-images.githubusercontent.com/10111024/161298097-77cb97e8-9550-465d-ab92-d74709b564d1.png)
#714

Reply:
Updated.